t/14-isotovideo.t

```diff
@@ -105,6 +105,33 @@ subtest 'isotovideo with git refspec specified' => sub {
             opts => "casedir=$data_dir/tests test_git_refspec=deadbeef _exit_after_schedule=1") } qr/Checking.*local.*deadbeef/, 'refspec in local git repository would be checked out';
 };
 
+subtest 'isotovideo with wheels' => sub {
+    chdir($pool_dir);
+    unlink('vars.json') if -e 'vars.json';
+    my $specfile = path("$data_dir/tests/wheels.yaml");
+
+    $specfile->spurt("wheels: [foo/bar]");
+    combined_like { isotovideo(
+            opts => "casedir=$data_dir/tests _exit_after_schedule=1") } qr@Invalid.*Missing property@, 'invalid YAML causes error';
+    $specfile->spurt("version: v99\nwheels: [foo/bar]");
+    combined_like { isotovideo(
+            opts => "casedir=$data_dir/tests _exit_after_schedule=1") } qr@Unsupported version@, 'unsupported version';
+
+    $specfile->spurt("version: v0.1\nwheels: [https://github.com/foo/bar.git]");
+    combined_like { isotovideo(
+            opts => "casedir=$data_dir/tests _exit_after_schedule=1") } qr@Cloning.*foo/bar@, 'single repo with full url';
+    $specfile->spurt("version: v0.1\nwheels: [https://github.com/foo/bar.git#branch]");
+    combined_like { isotovideo(
+            opts => "casedir=$data_dir/tests _exit_after_schedule=1") } qr@Cloning.*XXXfoo/bar@, 'single repo with branch';
+    $specfile->spurt("version: v0.1\nwheels: [foo/bar]");
+    combined_like { isotovideo(
+            opts => "casedir=$data_dir/tests _exit_after_schedule=1") } qr@Cloning.*foo/bar@, 'single repo';
+    $specfile->spurt("version: v0.1\nwheels: [foo/bar, spam/eggs]");
+    combined_like { isotovideo(
+            opts => "casedir=$data_dir/tests _exit_after_schedule=1") } qr@Cloning.*spam/eggs@, 'two repos';
+    $specfile->remove;
```

Comment:
How much does this increase the runtime? Please compare numbers from before and afterwards. The goal should be to have no significant increase and prefer to reduce test runtimes where possible to reach tests that run well below 1s.

Reply:
I don't know. Applying TDD means I'm adding tests first to be sure things will work and this is how existing tests were written.

Reply:
I don't really understand what you mean by that. I suggest you call `prove --timer -I. t/14-isotovideo.t` with and without your changes (e.g. after `git stash` to stash your changes) and compare the runtime.

Reply:
I mean that I added *new* tests and they're not even passing. How would you reduce test runtime in that context? Are you seeing any obvious issues with how I'm testing this?

Reply:
I don't see any obvious issues yet. Of course you can focus first on getting tests to work. I am just pointing out what we also should look out for. Currently if I call `prove -I. t/14-isotovideo.t` locally it takes 13s and we should be careful to not extend that (significantly).